app/routes/git_tokens.py

```
@@ -0,0 +1,99 @@
+    
+    Returns:
+        A list of dictionaries containing token metadata and masked token values.
+    """
+    try:
+        client = SupabaseClient()
+        res = client.select(table="git_label", columns="label, id, git_hosting,token_value, created_at")
+
+        formatted_tokens = [t for t in (format_token_response(token) for token in res) if t]
+        return formatted_tokens
+
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)